mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/138 - component network tab

Closes #138 

To test:
- Run any scenario
- Open Components View and click Network tab
- Select computer and network (computer must be connected to at least one network)
- Make sure ACL rights list appear for possible users and groups
- Make sure ACL rights can be modified via checkboxes or cleared with X button that appears when any permissions are set
